common/helpers/dayjs.js

```diff
@@ -12,13 +12,19 @@ import utc from 'dayjs/plugin/utc'
 import customParseFormat from 'dayjs/plugin/customParseFormat'
 import localizedFormat from 'dayjs/plugin/localizedFormat'
 import isBetween from 'dayjs/plugin/isBetween'
+import isSameOrBefore from 'dayjs/plugin/isSameOrBefore'
+import isSameOrAfter from 'dayjs/plugin/isSameOrAfter'
 import duration from 'dayjs/plugin/duration'
 import formatDatePeriod from './dayjs/formatDatePeriod.js'
+import timezone from 'dayjs/plugin/timezone'
 
 dayjs.extend(utc)
+dayjs.extend(timezone)
```

Comment:
Do we really use this extension? Everything should be done with utc now, shouldn't it?

Reply:
Not really, but it makes it easier to always use the dayjs.tz object which uses the changed timezone instead of the dayjs object.
so i would leave it here.